dmoj/settings.py

```diff
@@ -98,6 +98,12 @@
 
 VNOJ_TAG_PROBLEM_MIN_RATING = 1900  # Minimum rating to be able to tag a problem
 
+# Cooldown time (minutes) between each comment
+VNOJ_COMMENT_COOLDOWN = 10
+
+# Cooldown time (minutes) between each blog post
+VNOJ_BLOG_COOLDOWN = 3 * 60
```

Comment:
I think this should be in seconds for more flexibility.

Also, 10 minutes between comments seems too restricting for me. Maybe reduce this to 1 minute?